lib/api/src/main/java/io/atlasmap/v2/ActionParameter.java

```diff
@@ -19,124 +19,124 @@
 
     /**
      * Gets the value of the values property.
-     * 
+     *
      * <p>
      * This accessor method returns a reference to the live list,
      * not a snapshot. Therefore any modification you make to the
      * returned list will be present inside the JAXB object.
      * This is why there is not a <CODE>set</CODE> method for the values property.
-     * 
+     *
      * <p>
      * For example, to add a new item, do as follows:
      * <pre>
      *    getValues().add(newItem);
      * </pre>
-     * 
-     * 
+     *
+     *
      * <p>
      * Objects of the following type(s) are allowed in the list
      * {@link String }
-     * 
-     * 
+     *
+     *
      */
     public List<String> getValues() {
         if (values == null) {
-            values = new ArrayList<String>();
+            values = new ArrayList<>();
         }
         return this.values;
     }
 
     /**
      * Gets the value of the name property.
-     * 
+     *
      * @return
      *     possible object is
      *     {@link String }
-     *     
+     *
      */
     public String getName() {
         return name;
     }
 
     /**
      * Sets the value of the name property.
-     * 
+     *
      * @param value
      *     allowed object is
      *     {@link String }
-     *     
+     *
      */
     public void setName(String value) {
         this.name = value;
     }
 
     /**
      * Gets the value of the displayName property.
-     * 
+     *
      * @return
      *     possible object is
      *     {@link String }
-     *     
+     *
      */
     public String getDisplayName() {
         return displayName;
     }
 
     /**
      * Sets the value of the displayName property.
-     * 
+     *
      * @param value
      *     allowed object is
      *     {@link String }
-     *     
+     *
      */
     public void setDisplayName(String value) {
         this.displayName = value;
     }
 
     /**
      * Gets the value of the description property.
-     * 
+     *
      * @return
      *     possible object is
      *     {@link String }
-     *     
+     *
      */
     public String getDescription() {
         return description;
     }
 
     /**
      * Sets the value of the description property.
-     * 
+     *
      * @param value
      *     allowed object is
      *     {@link String }
-     *     
+     *
      */
     public void setDescription(String value) {
         this.description = value;
     }
 
     /**
      * Gets the value of the fieldType property.
-     * 
+     *
      * @return
      *     possible object is
      *     {@link FieldType }
-     *     
+     *
      */
     public FieldType getFieldType() {
         return fieldType;
     }
 
     /**
      * Sets the value of the fieldType property.
-     * 
+     *
      * @param value
      *     allowed object is
      *     {@link FieldType }
-     *     
+     *
```

Comment:
These white space changes make this already big PR even bigger.  I'd recommend reviewing your change set next time and dropping these types of changes.